Gemfile.lock

```diff
@@ -422,4 +415,4 @@ DEPENDENCIES
   whenever
 
 BUNDLED WITH
-   1.10.5
+   1.10.6
```

Comment:
I would :heart: to link this thing to a required version of bundler in the Gemfile itself. There have been a lot of discussion in the bundler repo about this version mark, and the final word is that devs should align their bundler version when possible.